### DISCUSSION TITLE
Shortcut/Hotkey support

### DISCUSSION THREAD

**Discussion by GreyManSoftware** [bkaankose/Wino-Mail#470] [Ideas]
The Windows Mail and Calendar has a bunch of hotkeys and shortcuts such as ctrl+return to send an email. It would be good to port over the most commonly used hotkeys to enhance the usability.

Thanks!

**Comment by GreyManSoftware** [comment]
Any thoughts on this? Seems like a really simple feature to add and would be a huge quality of life improvement!

**Comment by bkaankose** [comment]
Yesterday I released 2.0 beta which introduces keyboard shortcuts. You can give that a try and let me know whether it works for you or not in the release discussion.
